index.html

```diff
@@ -835,9 +872,11 @@ <h3>
                 <a data-cite=
                 "resource-timing-2#dom-performanceresourcetiming-nexthopprotocol">
                 <code>nextHopProtocol</code></a>, <a data-link-for=
-                "PerformanceNavigationTiming">unloadEventStart</a> and
+                "PerformanceNavigationTiming">unloadEventStart</a>,
+                <a data-link-for=
+                "PerformanceNavigationTiming">unloadEventEnd</a> and
                 <a data-link-for=
-                "PerformanceNavigationTiming">unloadEventEnd</a>. Set
+                "PerformanceNavigationTiming">workerStart</a>. Set
                 <a data-cite=
```

Comment:
In line 884 below, is it still correct to go to fetch-start-step or is it worker-start-step?

Reply:
`fetch-start-step` is correct -- otherwise `workerStart` would be overwritten and thus the "**last**" request in the final same-origin redirect chain instead of the "**first** request in final same-origin redirect chain" like we want.